install-online/profile.hbs.md

```diff
@@ -350,6 +360,9 @@ Images are written to `SERVER-NAME/REPO-NAME/workload-name`. Examples:
     - Harbor has the form `repository: "my-project/supply-chain"`.
     - Docker Hub has the form `repository: "my-dockerhub-user"`.
     - Google Cloud Registry has the form `repository: "my-project/supply-chain"`.
+- `EXTERNAL-REGISTRY-FOR-LOCAL-SOURCE` is where local source artifacts are stored in the registry when `--source-image` is not provided during workload creation via Tanzu CLI. This is represented as <registry-server>/<repository-path>. For more information, see [Local Source Proxy](../local-source-proxy/install.hbs.md) 
```

Comment:
Should we simplify this to not mention `--source-image` and just say:

`EXTERNAL-REGISTRY-FOR-LOCAL-SOURCE` is where developer's local source is uploaded during inner-loop iterative development using Local source proxy during workload creation via Tanzu CLI.

Reply:
I would go with a slight modified verbiage as:
`EXTERNAL-REGISTRY-FOR-LOCAL-SOURCE` is where developer's local source is uploaded when using Local source proxy for workload creation via Tanzu CLI.